packages/medusa/src/strategies/batch-jobs/order/export.ts

```diff
@@ -36,6 +39,11 @@ class OrderExportStrategy extends AbstractBatchJobStrategy<OrderExportStrategy>
   protected readonly fileService_: IFileService<any>
   protected readonly batchJobService_: BatchJobService
   protected readonly orderService_: OrderService
+  protected readonly featureFlagRouter_: FlagRouter
+
+  protected readonly orderExportPropertiesDescriptors = [
```

Comment:
question: why are we copying this here?

Reply:
To avoid to mutate the original array in the index